bin/get-reads-alt-unmap.sh

```diff
@@ -6,27 +6,37 @@ set -eu -o pipefail
 Filter reads from a .bam file for HLA typing.\n
 Usage: ./get-reads-alt-unmap.sh <input.bam> <output.bam>"
 
-IN=$1
-OUT=$2
-NCORE=`grep -c '^processor' /proc/cpuinfo`
+IN="$1"
+OUT="$2"
+NCORE=$(grep -c '^processor' /proc/cpuinfo)
+# POSIX compliant system memory query
+MEM=$(awk 'BEGIN{for (i=1; i<ARGC;i++)
+   printf "%.0f\n", ARGV[i]}' "$(( $(grep MemTotal /proc/meminfo | awk '{print $2}') / 1000 * 0.95))")
```

Comment:
All good except that I am still getting this error here:
```line 15: 61837244 / 1000 * 0.95: syntax error: invalid arithmetic operator (error token is ".95")```

Somehow my shell does not allow evaluation of non-integer arithmetic within ```$(( ))```. The only way I can get the calculation through is to put it inside the `awk` clause.